styles/app.scss

```diff
@@ -1,6 +1,7 @@
 @import "~tailwindcss/base";
 @import "~tailwindcss/components";
 @import "~tailwindcss/utilities";
+@import 'react-phone-input-2/lib/style.css';
```

Comment:
If we're overriding styles do we need this

Reply:
Yes we need it otherwise i have to redesign the whole component. 
Reason why i included node_module into the path. Maybe there's other ways around it.

Reply:
Can't include node_modules in the path. That would significantly increase bundle size

Reply:
node_modules has been removed from the path.